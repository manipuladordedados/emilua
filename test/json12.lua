local json = require('json')

local o = {}
o.o = o
print(json.encode(o))
