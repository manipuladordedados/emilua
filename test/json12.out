Main fiber from VM 0x1 panicked: 'Reference cycle exists'
stack traceback:
	[string "?"]: in function ''
	[string "?"]: in function ''
	[string "?"]: in function 'encode'
	input.lua:5: in main chunk
	[C]: in function ''
	[string "?"]: in function <[string "?"]:0>
